.editorconfig

```diff
@@ -260,6 +260,9 @@ dotnet_diagnostic.IDE0059.severity = warning
 # IDE0060: Remove unused parameter
 dotnet_diagnostic.IDE0060.severity = warning
 
+# CA1012: Abstract types should not have constructors
```

Comment:
```suggestion
# CA1012: Abstract types should not have public constructors
```

Reply:
Thanks @mavasani. Opened https://github.com/dotnet/docs/pull/23629 to fix that in docs as well.